misk-crypto/src/main/kotlin/misk/crypto/pgp/RealPgpDecrypter.kt

```diff
@@ -0,0 +1,70 @@
+package misk.crypto.pgp
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+import org.bouncycastle.openpgp.PGPCompressedData
+import org.bouncycastle.openpgp.PGPEncryptedDataList
+import org.bouncycastle.openpgp.PGPException
+import org.bouncycastle.openpgp.PGPLiteralData
+import org.bouncycastle.openpgp.PGPOnePassSignatureList
+import org.bouncycastle.openpgp.PGPPrivateKey
+import org.bouncycastle.openpgp.PGPPublicKeyEncryptedData
+import org.bouncycastle.openpgp.PGPUtil
+import org.bouncycastle.openpgp.jcajce.JcaPGPObjectFactory
+import org.bouncycastle.openpgp.operator.jcajce.JcePublicKeyDataDecryptorFactoryBuilder
+import org.bouncycastle.util.io.Streams
+import java.security.Security
+
+internal class RealPgpDecrypter(private val privateKey: PGPPrivateKey) : PgpDecrypter {
+  init {
+    Security.addProvider(BouncyCastleProvider())
```

Comment:
I'd move this statement to `MiskCryptoModule` where the rest of the Tink primitives are initialized.